tests/rptest/services/redpanda.py

```diff
@@ -307,6 +307,20 @@ def should_compile(
         allow_list_element, AllowLogsOnPredicate)
 
 
+def in_fips_environment() -> bool:
+    """
+    Returns True if the file /proc/sys/crypto/fips_enabled is present and
+    contains '1', otherwise returns False.
+    """
+    fips_file = "/proc/sys/crypto/fips_enabled"
+    if os.path.exists(fips_file) and os.path.isfile(fips_file):
+        with open(fips_file, 'r') as f:
+            contents = f.read().strip()
+            return contents == '1'
```

Comment:
question (non-blocking): this check slightly differs from the one in redpanda source (here: asserting on the entire contents of the file, there: looking at only the first byte). it's a procfs file, so I'd think not, but is that a meaningful distinction?

Reply:
I don't believe so.  The `/proc/sys/crypto/fips_enabled` endpoint should only return '0' or '1'

Reply:
cool, just noticing